### ISSUE TITLE
libpng warning:

### ISSUE THREAD

**Issue by ghost** [friendica/friendica-directory#18]
libpng warning: Interlace handling should be turned on when using png_read_image

**Comment by MrPetovan** [comment]
What version are you running? The new one (version 2 onward) or the legacy `dir` project?

**Comment by ghost** [comment]
The new one 2.1.0

**Comment by MrPetovan** [comment]
I can't find any resource online about this error pertaining to a specific PHP code snippet, Can you please check your libpng version?

**Comment by ghost** [comment]
identify -list format | grep PNG
      JNG* PNG       rw-   JPEG Network Graphics
      MNG* PNG       rw+   Multiple-image Network Graphics (libpng 1.6.34)
      PNG* PNG       rw-   Portable Network Graphics (libpng 1.6.34)
           See http://www.libpng.org/ for details about the PNG format.
    PNG00* PNG       rw-   PNG inheriting bit-depth, color-type from original if possible
    PNG24* PNG       rw-   opaque or binary transparent 24-bit RGB (zlib 1.2.11)
    PNG32* PNG       rw-   opaque or transparent 32-bit RGBA
    PNG48* PNG       rw-   opaque or binary transparent 48-bit RGB
    PNG64* PNG       rw-   opaque or transparent 64-bit RGBA
     PNG8* PNG       rw-   8-bit indexed with optional binary transparency
root@mail:~#

**Comment by MrPetovan** [comment]
Thanks, there was that one issue that was supposed to be solved in libpng 1.5.something, but you already are at version 1.6.

I've got nothing. 🤷‍♂️

**Comment by AndyHee** [comment]
`libpng warning: Interlace handling should be turned on when using png_read_image`

I can confirm this warring that's clogging up my log too. I'm on libpng 1.6.28 version.

**Comment by AndyHee** [comment]
I'm looking at the discussions here: https://sourceforge.net/p/libpng/bugs/237/
the warning might actually relate to PHP.

**Comment by MrPetovan** [comment]
Yeah, I feel like we've been doing image manipulation wrong for a while.

**Comment by AndyHee** [comment]
OK, no worries.
